app/src/main/java/org/gnucash/android/ui/transaction/TransactionFormFragment.java

```diff
@@ -745,15 +741,15 @@ private void saveNewTransaction() {
 
         mAccountsDbAdapter.beginTransaction();
```

Comment:
`#beginTransaction()` works on the whole database, not on specific tables. So it makes no difference.